.dockerignore

```diff
@@ -1,5 +1,6 @@
 **/node_modules/
-/.git/
+# TODO: separate .dockerignore for RHTAP?
+# /.git/ <- required to pass .git to the roxctl container for avoiding "Malformed status.sh output line STABLE_GIT_SHORT_SHA"
```

Comment:
Right, `status.sh` is used in builds and it requires git history. We need to make it visible to RHTAP. We need to solve that .dockerignore problem.

Reply:
To be that seems to an easy implementation (_just_ a parameter with a default value to pass down to buildah). I'll join the office hours and ask for a starting point to contribute this.

Reply:
Provided PR https://github.com/redhat-appstudio/build-definitions/pull/615